### ISSUE TITLE
Warlords of Draenor Timewalking gear is considered scrap

### ISSUE THREAD

**Issue by BarbokOL** [Jaliborc/Scrap#57]
Basically all in the title, it considers the warlords timewalking gear as scrap and tries to autosell it if the option to sell old gear is enabled, even when the item level is over 370 

In my case the specific item is "Rivet-Sealed Shoulderplates (Timewarped Warforged)
![image](https://user-images.githubusercontent.com/42697151/58748389-53e57780-8478-11e9-8792-a31d266d0880.png)

**Comment by martijnhartlief** [comment]
Yup even 415 timewalking gear is sold with the "old gear" option enabled
